index.html

```diff
@@ -46,7 +46,7 @@ <h3 class="community-organizer">Community Organizer</h3>
 
 		<h3 class="moderator">Discord Moderators</h3>
 		<p>The moderators are in charge of making sure everything stays civil, spam bots get properly cleaned up, and everyone is acting like a reasonably mature, responsible human being. Our active* mods on Discord are:</p>
-		<p><a href="#masterofgrey">MasterOfGrey</a> • <a href="#nana_scone">Nana Scone</a> • <a href="#warlord_mike">Warlord Mike</a> • <a href="#warp_core">warp_core</a></p>
+		<p><a href="#nana_scone">Nana Scone</a> • <a href="#warp_core">warp_core</a></p>
```

Comment:
```suggestion
		<p><a href="#masterofgrey">MasterOfGrey</a> • <a href="#nana_scone">Nana Scone</a> • <a href="#warlord_mike">Warlord Mike</a> • <a href="#warp_core">warp_core</a></p>
```

Mike responded pretty fast to Darcy leaving (albeit not in public channels), Grey still pops up every once in a while and also seems to respond to pings, and they're both still on the official list of mods, so I think it might be best to keep them for now.